### PR TITLE
Update Create Your First REST API tutorial

### PR DESCRIPTION
**Pull request by RukshiW**
Update Create Your First REST API tutorial

### CODE REVIEW

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 1]
```diff
@@ -48,78 +52,85 @@ Let's create a new REST API component as follows:
...
+    The low-code diagram of this resource opens.
 
-    1. Select **Connector** and then select **COVID-19 by ballerinax**.
+4. To connect to the COVID-19 API and retrieve data, follow the sub-steps below:
 
-    2. In the **Covid19 API Connection**  and click **Continue to Invoke API**.
+    1. Click **Connector** to add a connector. Then search for the **COVID-19 by ballerinax** connector, and click on it once it appears in the search results.
 
-    3. In the **Operation** drop-down list, select **getStatusByCountry** and enter details as follows in the other fields:
+    2. In the **Connector** panel, `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:100]:
```suggestion
    2. In the **Connector** panel, enter `covid19Client` as the **Connection Name** and click **Continue to Invoke API**.
```

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 2]
```diff
@@ -158,33 +169,31 @@ Let's design the REST API as follows:
...
 
-        Click the last **+** icon in the low-code diagram and click **Return**.
+    In the **Return Expression** field, enter `payload`.
 
-        In the **Return Expression** field, enter `payload`.
-
-        Click **Save**.
+    Click **Save**.
 
 Now you have completed designing the `CovidStatus` service.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:192]: `CovidStatus` service is not correct, right? Should be "`Statistics` REST API" isn't it?

File: en/docs/tutorials/create-your-first-rest-api.md [hunk 3]
```diff
@@ -158,33 +169,31 @@ Let's design the REST API as follows:
...
-         | **Variable Type** | `json`                        |
-         | **Variable Name** | `payload`                     |
-         | Expression        | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
+    Click **Save**.
 
-    2. Click **Save**.
+10. To respond with the JSON payload, add a `Return` statement.
 
-    3. To respond with the JSON payload, add a `Return` statement.
+    Click the last **+** icon in the low-code diagram and click **Return**.
```
Comment (Nashaath) [inline en/docs/tutorials/create-your-first-rest-api.md:186]: Should we include numbering for the sub-steps?

Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Done

File: en/docs/tutorials/create-your-first-rest-api.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/tutorials/create-your-first-rest-api.md]: Corrected.

**Review by Nashaath — APPROVED** [review]

**Pull request merged**